package.json

```diff
@@ -1,69 +1,253 @@
+    "content-disposition": "^0.5.2",
+    "content-type": "^1.0.4",
+    "cookie": "^0.3.1",
     "cookie-parser": "^1.4.3",
+    "cookie-signature": "^1.1.0",
+    "cross-env": "^5.2.0",
+    "encoder": "^0.2.4",
+    "encodeurl": "^1.0.2",
+    "escape-html": "^1.0.3",
+    "eslint": "^5.6.1",
+    "eslint-config-prettier": "^3.1.0",
+    "eslint-config-xo-lass": "^1.0.3",
+    "eslint-plugin-compat": "^2.5.1",
+    "eslint-plugin-prettier": "^3.0.0",
+    "etag": "^1.8.1",
     "express": "^4.16.3",
     "express-session": "^1.15.6",
+    "fixpack": "^2.3.1",
+    "fresh": "^0.5.2",
+    "husky": "^1.1.0",
+    "lint-staged": "^7.3.0",
     "marked": "^0.5.0",
     "mocha": "^3.5.3",
     "multer": "^1.3.0",
+    "nyc": "^13.0.1",
+    "on-finished": "^2.3.0",
+    "parseurl": "^1.3.2",
+    "path-is-absolute": "^1.0.1",
+    "prettier": "^1.14.3",
+    "proxy-addr": "^2.0.4",
+    "range-parser": "^1.2.0",
+    "remark-cli": "^5.0.0",
+    "remark-preset-github": "^0.0.8",
+    "rimraf": "^2.6.2",
+    "safe-buffer": "^5.1.2",
+    "send": "^0.16.2",
     "should": "^13.2.0",
     "should-http": "^0.1.1",
+    "statuses": "^1.5.0",
+    "tag": "^0.4.1",
+    "tinyify": "^2.4.3",
+    "type-is": "^1.6.16",
+    "utils-merge": "^1.0.1",
+    "vary": "^1.1.2",
+    "xo": "^0.23.0",
     "zuul": "^3.12.0"
   },
-  "browser": {
-    "./lib/node/index.js": "./lib/client.js",
-    "./test/support/server.js": "./test/support/blank.js"
+  "engines": {
+    "node": ">=8.8.1"
   },
-  "component": {
-    "scripts": {
-      "superagent": "lib/client.js"
+  "files": [
+    "lib",
+    "dist"
+  ],
+  "homepage": "https://github.com/visionmedia/superagent",
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged && npm test",
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
     }
   },
-  "main": "./lib/node/index.js",
-  "engines": {
-    "node": ">= 6.0"
+  "jsdelivr": "dist/superagent.min.js",
+  "keywords": [
+    "ajax",
+    "api",
+    "cancel",
+    "client",
+    "http",
+    "https",
+    "promise",
+    "promises",
+    "request",
+    "requests",
+    "response",
+    "rest",
+    "retry",
+    "superagent",
+    "timeout",
+    "transform",
+    "xhr",
+    "xmlhttprequest"
+  ],
+  "license": "MIT",
+  "lint-staged": {
+    "linters": {
+      "*.js": [
+        "prettier --write",
+        "git add"
+      ],
+      "*.md,!CONTRIBUTING.md,!HISTORY.md,!docs": [
+        "remark . -qfo",
+        "git add"
+      ],
+      "package.json": [
+        "fixpack",
+        "git add"
+      ]
+    }
+  },
+  "main": "lib/node/request.js",
+  "nyc": {
+    "reporter": [
+      "lcov",
+      "html",
+      "text"
+    ]
+  },
+  "prettier": {
+    "singleQuote": true,
+    "bracketSpacing": true,
+    "trailingComma": "none"
+  },
+  "remarkConfig": {
+    "plugins": [
+      "preset-github"
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/visionmedia/superagent"
+  },
+  "scripts": {
```

Comment:
I use `fixpack` which organizes them differently, I think there might be a way to configure the order somehow with `fixpack`.  Is there a better alternative than using fixpack?

Reply:
It seems like what I'm looking for should be the default behavior: From [the `fixpack` docs](https://www.npmjs.com/package/fixpack):

> It will re-write your package.json file as follows:
>
> - name first
> - description second
> - version third
> - author fourth
> - all other keys in alphabetical order
> - dependencies and devDependencies sorted alphabetically
> - newline at the end of the file

It seems you could create a `.fixpackrc` file to manage the configuration.

Personally I just manage `package.json` files manually (save for `npm install` runs)